cms/db/contest.py

```diff
@@ -542,7 +524,7 @@ def tokens_available(self, username, task_name, timestamp=None):
         future. Also, if r[0] == 0 and r[1] is None, then r[2] should
         be ignored.
 
-        username (string): the username of the user.
+        participation (Participaiton): the participation.
         task_name (string): the name of the task.
```

Comment:
Since we pass a Participation object, shouldn't we pass a Task object instead of a task_name string?